refine

```diff
@@ -657,7 +658,7 @@ broker_appengine_run() {
 findbugs() {
     findbugs_prepare
     
-    MVN_PARAMS="-Dfindbugs.dir=${REFINE_TOOLS_DIR}/${FINDBUGS_DIR}"
+    MVN_PARAMS="-Dfindbugs.dir=${REFINE_TOOLS_DIR}/${FINDBUGS_DIR}"f
```

Comment:
Is this "f" really meant to be there?

Reply:
No no, that was by mistake. I'll amend that in the new PR